graph-chat.md

```diff
@@ -19,16 +19,16 @@ TODO
 
 ## Duplex connection
 
-Majority of chat scenarios requires duplex (bi-directional) connections between participants. Graph-chat protocol uses multiple unidirectional connections (graph edges) created on multiple edge-messaging servers to implement duplex connections.
+Majority of chat scenarios requires duplex (bi-directional) connections between participants. Graph-chat protocol uses multiple unidirectional connections (simplexes) created on multiple simplex messaging servers to implement duplex connections.
 
```

Comment:
Not sure we need to name them - can we just drop? We can just call them "simplex connections" or where it needs to be clarified in the beginning - "simplex (unidirectional) connections". Previously it was to justify the name really.